### ISSUE TITLE
Yaks don't shoot at targets below them while they're circling

### ISSUE THREAD

**Issue by deniz1a** [OpenRA/OpenRA#7473]
They just keep circling.

**Comment by obrakmann** [comment]
On which release is that? That should have been fixed with #7088.

**Comment by obrakmann** [comment]
```
[19:52:29]	<antares79> deniz1a: on which release did you encounter the problem with the circling yaks
[19:52:30]	<antares79> ?
[19:52:38]	<deniz1a> latest release
[19:52:43]	<deniz1a> current one
[19:52:48]	<antares79> not the playtest?
[19:52:50]	<deniz1a> no 
```
fixed by #7088
